stylesheets/relatedProjects.css

```diff
@@ -195,17 +195,18 @@ a.tutorial-link:hover {
 		marg-left: -3%;
 	}
 	.project-image {
-		width: 80px;
-		height: 80%;
+		width: 600px;
+		height: 120%;

```

Comment:
this causes the image to be distorted, please check that the image dimensions stay the same also on phone, now the width is smaller than the height